tensorflow/python/keras/engine/training_distributed_v1.py

```diff
@@ -386,8 +386,8 @@ def _test_step_fn(inputs):
     try:
       _, batch_outs = K.batch_get_value([test_op, output_tensors])
     except errors.OutOfRangeError:
-      warning_msg = 'Make sure that your dataset can generate at least '
```

Comment:
Hi @bhack I would recommend to keep the quotes otherwise you will have additional spaces in the string because the indentation is part of the string.

```python
def without_quotes():
    warning_msg = 'Make sure that your dataset can generate at least \
    `steps` batches (in this case, {} batches).'.format(42)
    print(warning_msg)


def with_quotes():
    warning_msg = 'Make sure that your dataset can generate at least ' \
    '`steps` batches (in this case, {} batches).'.format(42)
    print(warning_msg)

without_quotes()
with_quotes()
```
will output
```
Make sure that your dataset can generate at least     `steps` batches (in this case, 42 batches).
Make sure that your dataset can generate at least `steps` batches (in this case, 42 batches).
```

Reply:
Yes, I didn't care of the ident as I was temp. on the mobile editor.